app/controllers/api/v1/ownership_certificate_validation_requests_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class OwnershipCertificateValidationRequestsController < Api::V1::ApplicationController
+      skip_before_action :verify_authenticity_token
+      before_action :check_token_and_set_application
+
+      def index
+        respond_to do |format|
+          format.json do
+            @ownership_certificate_validation_requests = @planning_application.ownership_certificate_validation_requests
+          end
+        end
+      end
+
+      def show
+        respond_to do |format|
+          if (@ownership_certificate_validation_request =
+                @planning_application.ownership_certificate_validation_requests.where(id: params[:id]).first)
+            format.json
+          else
+            format.json do
+              render json: {message: "Unable to find ownership certificate validation request with id: #{params[:id]}"},
+                status: :not_found
+            end
+          end
+        end
+      end
+
+      def update
+        @ownership_certificate_validation_request =
+          @planning_application.ownership_certificate_validation_requests.where(id: params[:id]).first
```

Comment:
Seems like this is the same finder as in `show`? Can we extract it to a before action hook and use `model.association.find(params[:id])`. Rather than using a conditional in the `respond_to` block you can use a `rescue_from` block to capture the Active Record exception and return the not found error JSON.